src/strings/csv/en.csv

```diff
@@ -602,6 +602,7 @@ NO_BATCHES_TO_WITHDRAW_FROM,There are no batches with withdrawable quantities.
 NO_DATA_YET,No data yet
 NO_FUTURE_DATES,No future dates allowed
 NO_INVENTORY_DESCRIPTION,"It looks like you haven’t added inventory yet. You can either import data from a CSV file, or add inventory using our inventory form."
+NO_MAP_DATA,No map data.
```

Comment:
We could create a ticket for @chudo76 to provide copy?

Reply:
I'll make a comment on the figma asking for this

Reply:
I'm going to take this out of strings and put a placeholder literal string here, so that we don't request translation for something that might change.

Reply:
sounds good 👍